modules/standard/IO.chpl

```diff
@@ -2728,6 +2729,9 @@ private proc _read_text_internal(_channel_internal:qio_channel_ptr_t,
     var ret = qio_channel_scan_string(false, _channel_internal, tx, len, -1);
     x = new string(tx, length=len, needToCopy=false);
     return ret;
+  } else if t == _bytes {
+    // handle _bytes
+    halt("Cannot scan into bytes objects yet");
```

Comment:
This is presumably temporary, but why `halt` here? You could set `err = EINVAL`.